src/Autodesk.Forge.DesignAutomation/ApiClient.cs

```diff
@@ -143,6 +149,28 @@ public async Task<List<T>> GetAllItems<T>(Func<string, Task<Page<T>>> pageGetter
             while (paginationToken != null);
             return ret;
         }
+
+        /// <summary>
+        /// Asynchronously retrieves a list of all items of type <typeparamref name="T"/> using the specified page getter function and parameter.
+        /// </summary>
+        /// <typeparam name="T">The type of the items to retrieve.</typeparam>
+        /// <typeparam name="P">The type of the parameter to pass to the page getter function.</typeparam>
+        /// <param name="pageGetter">A function that retrieves a page of items of type <typeparamref name="T"/> given a parameter of type <typeparamref name="P"/> and a pagination token.</param>
+        /// <param name="parameter">The parameter to pass to the page getter function.</param>
+        /// <returns>A task that represents the asynchronous operation. The task result contains a list of all items of type <typeparamref name="T"/>.</returns>
+        public async Task<List<T>> GetAllItems<T, P>(Func<P, string, Task<Page<T>>> pageGetter, P parameter)
+        {
+            var ret = new List<T>();
+            string paginationToken = null;
+            do
+            {
+                var resp = await pageGetter(parameter, paginationToken);
+                paginationToken = resp.PaginationToken;
+                ret.AddRange(resp.Data);
+            }
+            while (paginationToken != null);
+            return ret;
+        }
```

Comment:
Could you also add unit tests for this new function? 

See the example: https://github.com/Autodesk-Forge/forge-api-dotnet-design.automation/blob/2672f04bf625c106ee5c5e742e710715db74bf30/tests/e2e/TestAppBundlesApi.cs#L61-L71

Reply:
Yes, but to test locally? Is using the default environment variable?